docs/sphinx/conf.py

```diff
@@ -74,7 +74,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "Dagster"
-copyright = "2019, Elementl, Inc"  # pylint: disable=redefined-builtin
+copyright = "2019, Elementl, Inc"  # noqa: A001
```

Comment:
should this date be updated?

Reply:
I don't think this shows up in the docs-- I tried searching and nothing comes up. It's just some boilerplate from the initial sphinx scaffold. So IMO no need to keep it up to date.